test/docTest.js.html

```diff
@@ -135,6 +144,14 @@
             assert.equal($('#find-element').find('#sub-find-2').size, 1);
           });
 
+          it("should find the closest elements", function() {
```

Comment:
@mottam 
well, I changed some of the double quotes I found but his file needs to be reformatted. There are lots of double quotes.